/**:
  ros__parameters:
    max_velocity: 0.2
    max_acceleration: 0.075
    max_yaw_velocity: 0.4
    max_yaw_acceleration: 0.3